common/models/user.js

```diff
@@ -604,8 +604,8 @@ module.exports = function(User) {
     };
 
     UserModel.setter.password = function(plain) {
-      if (typeof plain !== 'string') {
-        return;
+      if (typeof plain === 'number') {
+        plain = plain.toString();
```

Comment:
Not necessary but could be cleaned up to one line as suggested in my other comment.